src/Page/Food/Builder.elm

```diff
@@ -581,7 +583,29 @@ updateIngredientFormView { excluded, db, ingredient, impact, transportImpact } =
             ]
             [ Icon.trash ]
         , span [ class "text-muted IngredientTransportLabel fs-7" ]
-            [ text "Transport pour cet ingrédient" ]
+            [ text "Transport pour cet ingrédient"
+            , if ingredient.byPlane /= Nothing then
+                label
+                    [ class "PlaneCheckbox ps-2" ]
+                    [ text "("
+                    , input
+                        [ type_ "checkbox"
+                        , class "form-check-input no-outline"
+                        , attribute "role" "switch"
+                        , checked <| ingredientQuery.byPlane == Just True
+                        , disabled <| ingredient.country == Nothing
+                        , onCheck
+                            (\checked ->
+                                event { ingredientQuery | byPlane = Just checked }
+                            )
+                        ]
+                        []
+                    , text " par avion)"
```

Comment:
Note: Still quite unsure about this UI, we could rather have a toggler using a plane icon or something. That may be addressed in a next PR though, where Laurent would be involved.

Reply:
Sure, it's part of the "todo list" in this PR 👍